### DISCUSSION TITLE
Vertical line displaced to all the way left of the document. Intended?

### DISCUSSION THREAD

**Discussion by eahso2000** [vslinko/obsidian-outliner#301] [Q&A]
Hello, does anyone have the issue, of when you stretch the app too far in the Obsidian Mac App, the text centers towards the middle of the screen, so do the bullet points, but the lines from "Outlines" stay to the sides, as if the app is still small
<img width="960" alt="Screenshot 2022-03-14 at 22 23 17" src="https://user-images.githubusercontent.com/75788219/158263691-a51bd171-50c4-47cd-9685-6149dd9e72f5.png">
.

**Comment by vslinko** [comment]
Hi @eahso2000 !

This is known issue https://github.com/vslinko/obsidian-outliner/issues/288
